client/verta/verta/_internal_utils/model_validator.py

```diff
@@ -81,11 +81,15 @@ def must_verta(model):
         )
 
     # model service passes __init__(artifacts) by keyword, so params must match
-    expected_init_params = tuple(getfullargspec(VertaModelBase.__init__).args)
-    init_params = tuple(getfullargspec(model.__init__).args)
+    expected_init_params = list(
+        inspect.signature(VertaModelBase.__init__).parameters.keys()
+    )
+    init_params = list(
+        inspect.signature(model.__init__).parameters.keys()
+    )
     if init_params != expected_init_params:
         raise TypeError(
-            "model __init__() parameters must be named {},"
+            "model __init__() parameters must be {},"
             " not {}".format(expected_init_params, init_params)
         )
```

Comment:
Changing this exception message is _technically_ an API breakage, but I'd consider it so unlikely for any user to be relying on this that it's fine.